src/min-reviewers.ts

```diff
@@ -0,0 +1,101 @@
+import { ReviewersAll, Label, PullContext, Review } from "./types";
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+
+export function getMinReviewers(labels: Label[]): ReviewersAll | number {
+  const pattern = /min-(?<number>\d|all)-reviewers/;
+
+  for (const label of labels) {
+    const m = label.name.match(pattern);
+    if (!m) {
+      continue;
+    }
+
+    const { groups = {} } = m;
+    const { number = "0" } = groups;
+    return number === "all" ? number : parseInt(number, 10);
+  }
+
+  return 0;
+}
+
+export function requirementPassed(
+  reviews: Review[],
```

Comment:
At first I thought why are you redefining reviews type, but then I tried to dig to it in @octokit packages myself...LOL :D

But anyway after more digging, found that there is a way to use it through deeper API:
https://www.npmjs.com/package/@octokit/openapi-types
![image](https://user-images.githubusercontent.com/20832646/210607512-7b24a964-3f90-4e6d-a9d1-99c978d01270.png)